src/app/pages/TopicPage/fixtures.js

```diff
@@ -0,0 +1,19 @@
+const rand = (min, max) => Math.floor(Math.random() * (max - min)) + min;
+const fixtureHeading = () =>
+  'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin sed lacus mi. In sed nunc non ante viverra aliquam. Quisque aliquam scelerisque eros, semper fermentum ipsum vehicula eu. Integer nec.'
+    .split(' ')
+    .slice(0, rand(5, 30))
+    .join(' ');
+const fixtureCat = () =>
+  `http://placekitten.com/${rand(200, 300)}/${rand(170, 250)}`;
+
+// eslint-disable-next-line import/prefer-default-export
+export const fixturePromos = () =>
+  new Array(23).fill().map((_, id) => ({
+    id,
+    heading: fixtureHeading(),
+    footer: '8th February 2022',
+    href: '#',
+    imageSrc: fixtureCat(),
+    imageAlt: 'evil monster',
```

Comment:
Let's be polite

Reply:
sorry, not following what you mean here - this alt text is a reference to the fact the image is of a cat - do you mean it's impolite to the cat?

either way, this will be replaced with with content from FABL, once we know what that looks like